app/views/services/_environment.html.haml

```diff
@@ -1,26 +1,26 @@
-%tr{ id: environment.environment_slug }
+%tr{ id: environment[:environment_slug] }
   %th
-    = ServiceEnvironment.find(environment.environment_slug).name
+    = ServiceEnvironment.find(environment[:environment_slug]).name
   %td
-    - if environment.url.present?
-      = link_to(environment.url, environment.url)
+    - if environment[:service_status][:url].present?
+      = link_to(environment[:service_status][:url], environment[:service_status][:url])
     - else
       \-
   %td
-    - if environment.status
-      %span{ class: "status status-#{environment.status.to_s.first}xx" }
-        = environment.status
+    - if !environment[:service_status][:status].nil?
```

Comment:
the `if !(thing).nil?` is better expressed as `if (thing).present?`, or even just `if (thing)`